src/opera/pge/cslc_s1/templates/OPERA_ISO_metadata_L2_CSLC_S1_template.xml.jinja2

```diff
@@ -1623,7 +2160,7 @@
                                         </eos:EOS_AdditionalAttributeDescription>
                                     </eos:reference>
                                     <eos:value>
-                                        <gco:CharacterString>{{ product_output.processing_information.s1_burst_metadata.range_window_coefficient }}{# ISO_OPERA_rangeWindowType #}</gco:CharacterString>
+                                        <gco:CharacterString>{{ product_output.processing_information.input_burst_metadata.range_window_coefficient }}{# ISO_OPERA_rangeWindowType #}</gco:CharacterString>
```

Comment:
# ISO_OPERA_rangeWindowType # is repeated here (probably should be ISO_OPERA_rangeWindowCoeffeicient or similar)